modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/rich/text/RichTextDDMFormFieldTemplateContextContributor.java

```diff
@@ -73,9 +73,19 @@
 		HttpServletRequest httpServletRequest =
 			ddmFormFieldRenderingContext.getHttpServletRequest();
 
+		String portletNamespace =
+			ddmFormFieldRenderingContext.getPortletNamespace();
+
+		String portletName = StringPool.BLANK;
```

Comment:
Every request has the `portletId` information that you're trying to extract from the `portletNamespace`. Could you please use the `ParamUtil` to get it, instead using `substring`? I believe you can work this one out by directly calling the following line in the `getEditorConfiguration` parameter:
`ParamUtil.getString(PortalUtil.getOriginalServletRequest(ddmFormFieldRenderingContext.getHttpServletRequest()), "p_p_id")`

Thx.

Reply:
@rodrigopaulino, thank you for this information! I have made the changes and will rerun the tests.
Thanks again!

Reply:
@rodrigopaulino It looks like the SF does not want to use `PortalUtil`. I will try to use portal service instead.

Reply:
Correct. Thanks! 😄